masterlist.yaml

```diff
@@ -10075,6 +10075,25 @@ plugins:
         itm: 6
         udr: 4
 
+  - name: 'HESU DiveRock.esp'
+    url:
+      - link: 'https://www.afkmods.com/index.php?/files/file/2344-the-hesu-mod-collection/'
+        name: 'The Hesu Mod Collection'
+    after:
+      - 'HeartOftheDead.esp'
+      - 'Secrets.esp'
+      - 'Servant of the Dawn.esp'
+    msg:
+      - <<: *patchProvided
+        subs: [ 'Secrets' ]
+        condition: 'active("Secrets.esp") and not file("Secrets-DiveRock Patch.esp")'
```

Comment:
Judging by your commit message, was there supposed to be a patch message for the SotD patch as well?